_data/projects.yml

```
@@ -274,6 +274,11 @@
   github: "https://github.com/killaitis/peloton"
   affiliate: true
   platforms: [jvm]
+- title: "perspective"
+  description: "Generic programming using higher kinded data and typeclasses. perspective provides tools for generic programming, and typeclasses for monad transformers and higher kinded data."
```

This reads a little repetitive to me. How does this sound? 
```suggestion
  description: "Provides tools for generic programming, and typeclasses for monad transformers and higher kinded data."
```